Add-ons/UmbracoCourier/Developer/DataResolvers.md

```diff
@@ -32,20 +32,20 @@ This is what dataresolvers do, add special meaning to specific data that matches
 ## Sample Data Resolver
 To show some code as fast as possible here is a commented code sample which outlines a simple resolver:
 
-	//inherit from ItemDataResolver and implement ResolvableTypes and ShouldExecute
+	// inherit from ItemDataResolver and implement ResolvableTypes and ShouldExecute
 	public class test : ItemDataResolverProvider
 	{
-	    //resolvableTypes are the types of content which can be processed by this resolver
-	    //all built-in classes are available under Umbraco.Courier.ItemProviders
-	    //this class handles all templates and nothing else
+	    // resolvableTypes are the types of content which can be processed by this resolver
+	    // all built-in classes are available under Umbraco.Courier.ItemProviders
+	    // this class handles all templates and nothing else
 	    public override List<Type> ResolvableTypes
 	    {
 	        get { return new List<Type> { typeof(Template) }; }
 	    }
-	 
-	    //ShouldExecute, a fast way to determine if the provider should trigger or not, under a specific event
-	    //in this case the resolver will only trigger during Packaging Event and if the template has any 
-	    //JavaScript resources packaged
+
+	    // ShouldExecute, a fast way to determine if the provider should trigger or not, under a specific event
+	    // in this case the resolver will only trigger during Packaging Event and if the template has any 

```

Comment:
@dampee 

I have a feeling it might be that you are seeing a GitHub wrapping issue, if you look on the main pull request page you see this:
![2018-06-28](https://user-images.githubusercontent.com/4398217/42040619-0dae12a0-7abe-11e8-8cb3-2ca419251a38.png)

But if you click to view the file changes you see this:

![2018-06-28 1](https://user-images.githubusercontent.com/4398217/42040618-0da1cfa4-7abe-11e8-9b22-6724b221540a.png)

Cheers, Chris